src/test/resources/org/broadinstitute/hellbender/tools/haplotypecaller/expected.testVCFMode.gatk4.vcf

```diff
@@ -26,161 +25,161 @@
 20	10002138	.	C	G	2443.77	.	AC=2;AF=1.00;AN=2;DP=53;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;MQ=56.38;QD=26.00;SOR=0.730	GT:AD:DP:GQ:PL	1/1:0,53:53:99:2472,169,0
 20	10002142	.	G	C	2407.77	.	AC=2;AF=1.00;AN=2;DP=52;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;MQ=56.31;QD=30.02;SOR=0.853	GT:AD:DP:GQ:PL	1/1:0,52:52:99:2436,166,0
 20	10002470	.	C	T	2522.77	.	AC=2;AF=1.00;AN=2;DP=61;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;MQ=48.62;QD=31.98;SOR=0.941	GT:AD:DP:GQ:PL	1/1:0,61:61:99:2551,189,0
 20	10002625	.	G	T	2665.77	.	AC=2;AF=1.00;AN=2;DP=64;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;MQ=59.05;QD=27.51;SOR=1.127	GT:AD:DP:GQ:PL	1/1:0,64:64:99:2694,193,0
 20	10003021	.	C	T	2383.77	.	AC=2;AF=1.00;AN=2;DP=60;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;MQ=58.99;QD=29.11;SOR=1.071	GT:AD:DP:GQ:PL	1/1:0,60:60:99:2412,181,0
-20	10003358	.	A	C	1454.77	.	AC=1;AF=0.500;AN=2;BaseQRankSum=0.004;ClippingRankSum=0.000;DP=90;ExcessHet=3.0103;FS=0.844;MLEAC=1;MLEAF=0.500;MQ=59.79;MQRankSum=1.011;QD=16.35;ReadPosRankSum=-0.444;SOR=0.572	GT:AD:DP:GQ:PL	0/1:45,44:89:99:1483,0,1477
+20	10003358	.	A	C	1454.77	.	AC=1;AF=0.500;AN=2;BaseQRankSum=0.004;DP=89;ExcessHet=3.0103;FS=0.844;MLEAC=1;MLEAF=0.500;MQ=59.79;MQRankSum=1.011;QD=16.35;ReadPosRankSum=-1.355;SOR=0.572	GT:AD:DP:GQ:PL	0/1:45,44:89:99:1483,0,1477
 20	10003651	.	T	C	3620.77	.	AC=2;AF=1.00;AN=2;DP=93;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;MQ=59.19;QD=28.08;SOR=1.511	GT:AD:DP:GQ:PL	1/1:0,93:93:99:3649,279,0
 20	10003692	.	A	G	2609.77	.	AC=2;AF=1.00;AN=2;DP=68;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;MQ=59.45;QD=23.23;SOR=1.352	GT:AD:DP:GQ:PL	1/1:0,68:68:99:2638,205,0
 20	10003832	.	G	A	2705.77	.	AC=2;AF=1.00;AN=2;DP=66;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;MQ=57.13;QD=29.03;SOR=1.473	GT:AD:DP:GQ:PL	1/1:0,66:66:99:2734,199,0
 20	10004094	.	A	C	1825.77	.	AC=2;AF=1.00;AN=2;DP=53;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;MQ=52.75;QD=34.45;SOR=1.524	GT:AD:DP:GQ:PL	1/1:0,53:53:99:1854,157,0
 20	10004147	.	A	G	1837.77	.	AC=2;AF=1.00;AN=2;DP=52;ExcessHet=3.0103;FS=0.000;MLEAC=2;MLEAF=1.00;MQ=54.99;QD=30.67;SOR=1.136	GT:AD:DP:GQ:PL	1/1:0,52:52:99:1866,156,0
-20	10004193	.	G	T	916.77	.	AC=1;AF=0.500;AN=2;BaseQRankSum=-0.392;ClippingRankSum=0.000;DP=52;ExcessHet=3.0103;FS=2.369;MLEAC=1;MLEAF=0.500;MQ=59.30;MQRankSum=3.385;QD=17.98;ReadPosRankSum=0.995;SOR=0.941	GT:AD:DP:GQ:PL	0/1:23,28:51:99:945,0,776
-20	10004222	.	C	CA	12.96	.	AC=1;AF=0.500;AN=2;BaseQRankSum=0.335;ClippingRankSum=0.000;DP=55;ExcessHet=3.0103;FS=1.494;MLEAC=1;MLEAF=0.500;MQ=60.05;MQRankSum=-2.042;QD=0.33;ReadPosRankSum=-0.052;SOR=0.434	GT:AD:DP:GQ:PL	0/1:31,8:39:50:50,0,923
-20	10004223	.	A	AG	47.73	.	AC=1;AF=0.500;AN=2;BaseQRankSum=-0.480;ClippingRankSum=0.000;DP=63;ExcessHet=3.0103;FS=0.000;MLEAC=1;MLEAF=0.500;MQ=58.33;MQRankSum=1.886;QD=1.02;ReadPosRankSum=2.234;SOR=0.638	GT:AD:DP:GQ:PL	0/1:36,11:47:85:85,0,982
```

Comment:
I spot checked this site and it looks pretty respectable.  I think this variant benefited from your changes.